inference-engine/tests/functional/inference_engine/lp_transformations/convolution_backprop_data_transformation.cpp

```diff
@@ -0,0 +1,314 @@
+        transform.transform(actualFunction);
+        std::shared_ptr<Node> refWeights = pass::low_precision::fold<opset1::Broadcast>(
+                testValues.expected.weights,
+                opset1::Constant::create(
+                        element::i64,
+                        Shape{inputShape.size()},
+                        Shape{inputShape[1], outputShape[1], 1, 1}));
+
+        if (!testValues.expected.transformed) {
+            refWeights = ngraph::builder::subgraph::ConvolutionBackpropDataFunction::getWeights(
+                outputShape,
+                netPrecision,
+                testValues.actual.fakeQuantizeOnWeights,
+                as_type_ptr<opset1::Constant>(refWeights));
+        } else {
+            refWeights = ngraph::builder::subgraph::ConvolutionBackpropDataFunction::getWeights(
+                outputShape,
+                netPrecision,
+                testValues.expected.dequantizationOnWeights,
+                as_type_ptr<opset1::Constant>(refWeights));
+        }
+
+        referenceFunction = ngraph::builder::subgraph::ConvolutionBackpropDataFunction::getReference(
+                testValues.expected.precisionBeforeDequantization,
+                netPrecision,
+                inputShape,
+                outputShape,
+                testValues.expected.dequantizationOnActivations,
+                refWeights,
+                testValues.expected.dequantizationAfter);
+    }
+
+    static std::string getTestCaseName(testing::TestParamInfo<ConvolutionBackpropDataTransformationParams> obj) {
+        const auto netPrecision = std::get<0>(obj.param);
+        auto inputShape = std::get<1>(obj.param);
+        ConvolutionBackpropDataTransformationTestValues testValues = std::get<2>(obj.param);
+
+        std::ostringstream result;
+        result << toString(testValues.params) << "_" <<
+               netPrecision << "_" <<
+               inputShape << "_" <<
+               testValues.actual.precisionBeforeDequantization << "_" <<
+               testValues.actual.dequantizationOnActivations << "_" <<
+               testValues.actual.dequantizationOnWeights << "_" <<
+               testValues.actual.fakeQuantizeOnWeights << "_" <<"_weights_" <<
+               testValues.actual.weights->get_element_type() << "_" << "{ " <<
+               testValues.actual.weights->cast_vector<float>()[0] << " }_";
+        return result.str();
+    }
+};
+
+TEST_P(ConvolutionBackpropDataTransformation, CompareFunctions) {
+    actualFunction->validate_nodes_and_infer_types();
+    auto res = compare_functions(referenceFunction, actualFunction, true, true, true);
+    ASSERT_TRUE(res.first) << res.second;
+}
+
+const std::vector<element::Type> netPrecisions = {
+        element::f32,
+        element::f16
+};
+
+const std::vector<ngraph::Shape> shapes = {
+        ngraph::Shape({ 1, 8, 16, 16 })
+};
+
+const std::vector<ConvolutionBackpropDataTransformationTestValues> testValues = {
```

Comment:
Do we need to add a test-case with `updatePrecisions=false` here?

Reply:
Added this case, thanks!